### PR TITLE
UPDATE UI_BP.tsv

### PR DESCRIPTION
**Pull request by faicciruz**
Unfortunately this update is not following a linear way because I'm working with RTT and making sure everything fits the game nicely. I will update the whole UI file eventually. 

In this PR I fixed some errors and added more translation.

I based this translation on previous PRs which were merged and then removed. These previous UI merges need to be reviewed, and this is what I intend to do in future updates.

### CODE REVIEW

**Comment by faicciruz** [comment]
Everything is tested in-game and fits nicely. I feel like updating and reviewing the whole file and then it up in multiples PR like it was done before.

File: UI_BP.tsv [hunk 1]
```diff
@@ -1535,7 +1535,7 @@ UI_20150428_001533	{@st45ty}Cor do Penteado
 UI_20150428_001534	{@st45ty}Titulo Disponível
 UI_20150428_001535	{@st41_yellow}Auxiliar de Habilidades
 UI_20150428_001536	{@st41b}Mostrar o nome de outros personagens{/}
-UI_20150428_001537	{@st43}Modo da UI{/}
+UI_20150428_001537	{@st43}Modo da Tela{/}
```
Comment (faicciruz) [inline UI_BP.tsv]: Interface* just a quick note to myself. I'll fix this when I get to my computer.

File: UI_BP.tsv [hunk 2]
```diff
@@ -1381,23 +1381,23 @@ UI_20150323_001379	{@sti6}Colocar ou retirar itens do armazém custa %s prata.{/
...
 UI_20150323_001385	{@st41}Cacelar Pré-Visualização
 UI_20150323_001386	{@st46}Minhas Moedas : %s{/}
 UI_20150323_001387	{@st41b}Alterar Alojamento{/}
-UI_20150323_001388	{@st59}Decorar alojamento{/}
+UI_20150323_001388	{@st59}Decorar Alojamento{/}
 UI_20150323_001389	{@st41b}Configurações do alojamento{/}
-UI_20150323_001390	{@st59}Alterar configurações do alojamento {/}
-UI_20150323_001391	{@st45ty}Cumprimentos do Personagem{/}
+UI_20150323_001390	{@st59}Alterar configurações do Alojamento{/}
+UI_20150323_001391	{@st45ty}Saudações do Personagem{/}
```
Comment (faicciruz) [inline UI_BP.tsv]: Saudação I think. I'll check this in a bit.

**Comment by imcgames** [comment]
Thank you :+1:

**Pull request merged**